src/test/SDL_test_common.c

```diff
@@ -1218,7 +1218,8 @@ SDLTest_CommonInit(SDLTest_CommonState *state)
             fullscreen_mode.format = SDL_PIXELFORMAT_RGB888;
             break;
         }
-        fullscreen_mode.refresh_rate = state->refresh_rate;
+        fullscreen_mode.refresh_rate.numerator = state->refresh_rate;
```

Comment:
The refresh rate should probably be changed to a float or double and the command line parameter should be changed to accept a floating point value.

Reply:
now a double
```

 433         state->refresh_rate = SDL_atof(argv[index]);
...
1221         fullscreen_mode.refresh_rate.numerator = (int) SDL_floor(state->refresh_rate * 1000000.0);
1222         fullscreen_mode.refresh_rate.denominator = 1000000;
1223 

```